src/dotnet-core-uninstall/Shared/Commands/UninstallCommandExec.cs

```diff
@@ -215,7 +216,7 @@ private static bool AskItAndReturnUserAnswer(IDictionary<Bundle, string> bundles
 
             var response = Console.ReadLine().Trim().ToUpper();
 
-            if (response.Equals("Y"))
+            if (response.Equals("Y") || response.Equals("YES"))
```

Comment:
Suggestion: We could dependency inject `Console.ReadLine().Trim().ToUpper();`, or extract method to make the `response` a pass-in parameter. So we can also test Y and Yes are acceptable answer.